mobile/native/src/logger.rs

```diff
@@ -28,8 +28,7 @@ pub fn log_base_directives(env: EnvFilter, level: LevelFilter) -> Result<EnvFilt
         // set to debug to show ldk logs (they're also in logs.txt)
         .add_directive("sled=warn".parse()?)
         .add_directive("bdk=warn".parse()?) // bdk is quite spamy on debug
-        .add_directive("lightning::ln::peer_handler=debug".parse()?)
-        .add_directive("lightning::chain=info".parse()?)
+        .add_directive("lightning=trace".parse()?)
```

Comment:
🔧 We are only printing debug logs to FLog. This change will increase the logs on the mobile app.

Reply:
Are you saying that this won't work because the app ignores trace logs?

Reply:
yes

Reply:
I guess we should change that? :) the app is the only consumer of this library, so we might as well have a single place to set the logs. 

of course the trace logs are also seen in the console when you're debugging locally from the app (whether it's native, simulator, or phone connected with usb), so the change makes sense anyway.

Reply:
Fixed here: https://github.com/get10101/10101/pull/826/commits/27f23d230796447b35729ca568b5c7c9f3b6fbd4.